modules/twinklespeedy.js

```diff
@@ -2087,6 +2109,11 @@ Twinkle.speedy.callback.evaluateUser = function twinklespeedyCallbackEvaluateUse
 		});
 	}
 
+	var requestsalt = false;
```

Comment:
Couldn't this bit simply be replaced with `var requestsalt = form.salting.checked || false;` or a ternary or something?

Reply:
Done